README.md

```diff
@@ -1,188 +1,52 @@
-# Ably Chat SDK for Android
+![Ably Chat Header](/images/ably-chat-github-header.png)
 
-<p style="text-align: left">
-    <img src="https://badgen.net/github/license/3scale/saas-operator" alt="License" />
-    <img src="https://img.shields.io/badge/version-0.3.0-2ea44f" alt="version: 0.3.0" />
-    <a href="https://github.com/ably/ably-chat-kotlin/actions/workflows/coverage.yml"><img src="https://img.shields.io/static/v1?label=coverage&message=80%2B%25&color=2ea44f" alt="coverage - 80+%"></a>
-</p>
+# Ably Chat Kotlin SDK
```

Comment:
I think we should call out Android here - as that's the main audience.

Perhaps

`Ably Chat SDK for Android and Kotlin` ?

Reply:
`Ably Chat Android and Kotlin SDK ` ??